app/Http/Controllers/Users/HomeController.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace App\Http\Controllers\Users;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\Post;
+use App\Models\Image;
+use App\Models\Category;
+use Illuminate\Support\Facades\DB;
+
+class HomeController extends Controller
+{
+    protected $pathToView = 'user.pages.';
+    protected $imgPosts;
+    protected $categoriesWithChildren;
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function __construct()
+    {
+        $this->imgPosts = Image::where('imageable_type', 'App\Models\Post')->get();
+        $this->categoriesWithChildren = Category::with('children')->whereNull('parent_id')->get();
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show()
+    {
+        //Lastest news
+        $latestPosts = Post::orderBy('created_at', 'desc')->take(5)->get();
+        //World news
+        $worldSubCategories = Category::with(['children'])->where('parent_id', 1)->get();
+        $worldPosts = $worldSubCategories->load('posts');
+        $worldPostsArray = array();
+        foreach ($worldSubCategories as $category) {
+            foreach ($category->posts as $post) {
+                $worldPostsArray[] = $post;
+            }
+        }
+        $worldPosts = array_slice($worldPostsArray, 0, 3);
```

Comment:
```suggestion
        $worldPosts = array_slice($worldPostsArray, config('model.posts.minDisplay'), config('model.posts.maxDisplay'));
```
bỏ 0 và 3 vào config nhé.
VD: config('model.posts.maxDisplay');